changelog.rst

```diff
@@ -15,6 +15,7 @@ Bug Fixes:
 * Fix the way we get host when using DSN (issue #765) (Thanks: `François Pietka`_)
 * Add missing keyword COLUMN after DROP (issue #769) (Thanks: `François Pietka`_)
 * Don't include arguments in function suggestions for backslash commands (Thanks: `Joakim Koljonen`_)
+* Fix TODO, optionally use POSTGRES_USER, POSTGRES_HOST POSTGRES_PASSWORD from environment (Thanks: `Dick Marinus`_)
```

Comment:
No need for "fix TODO" part. "Optionally use POSTGRES_USER, POSTGRES_HOST, POSTGRES_PASSWORD from environment" is good enough. :)